test/cipherlist_test.c

```diff
@@ -215,9 +215,41 @@ static int test_default_cipherlist_explicit(void)
     return result;
 }
 
+/* SSL_CTX_set_cipher_list() should fail if it clears all TLSv1.2 ciphers. */
+static int test_default_cipherlist_clear(void)
+{
+    SETUP_CIPHERLIST_TEST_FIXTURE();
+    SSL* s;
+    if (fixture == NULL)
+        return 0;
+
+    result = TEST_int_eq(SSL_CTX_set_cipher_list(fixture->server,
```

Comment:
`result` is not declared in this function. If it were it should be initialized to 0; and then, set to 1 just before the `end:` label

Reply:
It's declared by the (pre-existing) `SETUP_CIPHERLIST_TEST_FIXTURE` macro.